src/sentry/static/sentry/app/views/performance/transactionSummary/index.tsx

```diff
@@ -26,6 +31,10 @@ import withProjects from 'app/utils/withProjects';
 
 import SummaryContent from './content';
 import {addRoutePerformanceContext, getTransactionName} from '../utils';
+import {
+  PERCENTILE as VITAL_PERCENTILE,
+  WEB_VITAL_DETAILS,
+} from '../realUserMonitoring/constants';
```

Comment:
Should this directory be renamed to either measurements or vitals? Doesn't necessarily need to be part of this pull request.

Reply:
Yes, I'll be doing that in a follow up to consolidate these names. 👍